modules/apps/client-extension/client-extension-web/src/main/java/com/liferay/client/extension/web/internal/type/deployer/CETDeployerImpl.java

```diff
@@ -200,6 +209,16 @@ private List<ServiceRegistration<?>> _deploy(
 					jsImportMapsEntryCET.getURL())));
 	}
 
+	private List<ServiceRegistration<?>> _deploy(ThemeCSSCET themeCSSCET) {
+		return Arrays.asList(
+			_bundleContext.registerService(
+				ThemeCSSCET.class, themeCSSCET,
+				HashMapDictionaryBuilder.put(
+					"external.reference.code",
+					themeCSSCET.getExternalReferenceCode()
+				).build()));
```

Comment:
nice! this way there's a full decoupling bweteen cet deploy and subsequent actions that the "real" deployer will take.